client/transaction-pool/graph/src/pool.rs

```diff
@@ -378,8 +378,13 @@ impl<B: ChainApi> Pool<B> {
 		let block_number = self.resolve_block_number(at)?;
 		let mut result = HashMap::new();
 
-		for xt in xts {
-			let (hash, validated_tx) = self.verify_one(at, block_number, xt, force).await;
+		for (hash, validated_tx) in
+			futures::future::join_all(
```

Comment:
tbh I'd prefer to assign the result to a separate variable to avoid this super long `for` block :) But it's a minor style thing.